.github/workflows/X-Reusable-VerifyInstrumentation.yml

```
@@ -68,6 +68,7 @@ jobs:
         with:
           path: /home/runner/work/newrelic-java-agent/newrelic-java-agent/newrelic-agent/build/newrelicJar/newrelic.jar
           key: ${{ github.run_id }}
+          fail-on-cache-miss: true
```

fail fast when agent cache misses.